lib/relay.js

```diff
@@ -33,10 +33,12 @@ function requestHandler(filterRules) {
     logger.debug(logContext, 'received request over HTTP connection');
     filters(req, (error, result) => {
       if (error) {
+        const reason = 'Request does not match any accept rule, blocking HTTP request';
+        error.reason = reason;
         logContext.error = error;
-        logger.info(logContext, 'no rule match, blocking HTTP request');
+        logger.info(logContext, reason);
         // TODO: respect request headers, block according to content-type
-        return res.status(401).send(error.message);
+        return res.status(401).send({message: error.message, reason});
```

Comment:
This changes the response type from plain text to `application/json`, will also affect response headers.

I'd leave this change out for now as it carries more impact to the rest of the system. If you want to improve logging, stick to the logging :)

Reply:
This is the request handler, I'd like this info to be sent back to Snyk so we could see this in logz.io, rather than generic `blocked`